### DISCUSSION TITLE
can you please explain the units/numbers here?

### DISCUSSION THREAD

**Discussion by P-I-Engineer** [bassamanator/Sovol-SV06-firmware#46] [Q&A]
i don't understand the % meaning. thanks
i.e.:
26%
100%

i had to reduce the feedrates for the purge line or else tpu would jam.
the purge and the line.

![image](https://user-images.githubusercontent.com/11393479/233851233-fd91e280-e715-4418-b465-99d391998618.png)

**Comment by bassamanator** [comment]
I can't teach you how to write macros in Klipper. There are many resources on this topic out there, search for `klipper macro tutorial`.

The speeds for the purge and the line are 3.33mm/s and 15mm/s, quite safe. What's the shore value for the TPU you're trying to print?

**Reply by P-I-Engineer** [reply]
AH! ok, i didn't realize the % were wrapped, these dang eyes.

it's a very low shore tpu, old overature tpu. anyway, i'm good now. thanks for the term to search. and thanks for all this setup, the printer is much better with klipper. 

i'd recommend getting rid of all the z34 gantry slamming stuff. the soup can method makes a lot more sense in comparing to what you're leveling to, the build plate.

**Reply by bassamanator** [reply]
Why would I remove functionality that is included as part of the official firmware, functionality that removes user error from the equation? How is this Klipper configuration improved by removing said functionality?
